### PR TITLE
CUMULUS-392: update queue-granules to return the sfn-execution-arns queued

### PR DESCRIPTION
**Pull request by jennyhliu**
**Summary:** Summary of changes

Addresses [CUMULUS-392: queue-granules not returning the sfn-execution-arns queued](https://bugs.earthdata.nasa.gov/browse/CUMULUS-392)

## Changes

* updated queue-granules to return the sfn-execution-arns queued and pdr if exists.
* added pdr to ingest message meta.pdr instead of payload, so the pdr information doesn't get lost in the ingest workflow, and ingested granule in elasticsearch has pdr name.
* fixed sf-sns-report schema, remove the invalid part

## Test Plan
Things that should succeed before merging.

- [x ] Unit tests
- [x] Adhoc testing: tested in AWS, will submit separate PR for integration-test
- [x ] Update CHANGELOG
- [ ] Run `./bin/eslint-ratchet` and verify that eslint errors have not increased
  - [ ] Commit `.eslint-ratchet-high-water-mark` if the score has improved

## Release PR

If this is a **release** PR, make sure you branch name starts with `release-` prefix, e.g. `release-v-1.1.2`.

Reviewers: @tag-your-friends

### CODE REVIEW

File: tasks/queue-granules/schemas/output.json [hunk 1]
```diff
@@ -2,8 +2,23 @@
...
-    "granules_queued": { "type": "integer" }
+    "running": {
+      "type": "array",
+      "description": "a list of step function execution arns of granules queued",
+      "items": {
+        "type": "string"
+      }
+    },
+    "pdr": {
+      "description": "Product Delivery Record.",
```
Comment (laurenfrederick) [inline tasks/queue-granules/schemas/output.json]: Remove the period here

File: packages/ingest/test/queue.js [hunk 2]
```diff
@@ -66,7 +66,8 @@ test('the queue receives a correctly formatted workflow message', async(t) => {
         },
```
Comment (laurenfrederick) [inline packages/ingest/test/queue.js]: Test that the arn is returned from line 48 and that it's in the correct format.

Comment (laurenfrederick) [reply packages/ingest/test/queue.js]: Also test that the pdr comes back in the expected message

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,10 +5,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
 ## [Unreleased]
+- **CUMULUS-392 "queue-granules not returning the sfn-execution-arns queued"**
+  - updated queue-granules to return the sfn-execution-arns queued and pdr if exists.
+  - added pdr to ingest message meta.pdr instead of payload, so the pdr information doesn't get lost in the ingest workflow, and ingested granule in elasticsearch has pdr name.
+  - fixed sf-sns-report schema, remove the invalid part
 
 ### Added
 - **CUMULUS-448** Add code coverage checking using [nyc](https://github.com/istanbuljs/nyc).
-
+update queue-granules to return the sfn-execution-arns queued and pdr if exists.
```
Comment (laurenfrederick) [inline CHANGELOG.md]: Remove this

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,10 +5,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- **CUMULUS-392 "queue-granules not returning the sfn-execution-arns queued"**
```
Comment (laurenfrederick) [inline CHANGELOG.md]: This section should go under an 'Updated' heading.

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,10 +5,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+  - updated queue-granules to return the sfn-execution-arns queued and pdr if exists.
+  - added pdr to ingest message meta.pdr instead of payload, so the pdr information doesn't get lost in the ingest workflow, and ingested granule in elasticsearch has pdr name.
+  - fixed sf-sns-report schema, remove the invalid part
```
Comment (laurenfrederick) [inline CHANGELOG.md]: This part should go under a fixed heading

File: CHANGELOG.md (hunk 3 above)
Comment (laurenfrederick) [inline CHANGELOG.md]: Remove this

File: packages/ingest/test/queue.js [hunk 6]
```diff
@@ -66,7 +66,8 @@ test('the queue receives a correctly formatted workflow message', async(t) => {
...
         payload: { granules: [granule] }
       };
-
+      t.not(actualMessage.cumulus_meta.execution_name);
```
Comment (laurenfrederick) [inline packages/ingest/test/queue.js]: Also test that the arn is returned from line 48 and that with a pdr the expected message is correct

File: tasks/queue-granules/schemas/output.json (hunk 1 above)
Comment (laurenfrederick) [inline tasks/queue-granules/schemas/output.json]: Remove the period

**Comment by jennyhliu** [comment]
addressed the comments.

**Pull request merged**